client/components/post-schedule/clock.jsx

```diff
@@ -23,10 +23,10 @@ const noop = () => {};
 /**
  * Check if the given value is useful to use in time format
  * @param {String} value - time value to check
- * @return {String} checked value
+ * @return {String} checked value or `false`
  */
 function checkTimeValue( value ) {
```

Comment:
Added in 5875bfa4b96655e5a2f87bfc099e60a9ceec5a28
I'm afraid I'll need an aditional review 😞